newsfragments/1588.misc.rst

```diff
@@ -0,0 +1 @@
+Added block_identifier parameter to `ContractConstructor.estimateGas` method.
```

Comment:
I think this counts as a feature! I usually use misc for things that don't impact any APIs for end users, like testing changes, etc. because anything with misc just shows up as a link, and it's nice to highlight things that have changed with the API.

Reply:
@kclowes That makes sense! And no problem. Happy to help :)